<launch>
    <node name="node_a" pkg="ir2425_group_08" type="node_a" output="screen" />
    <node name="node_b" pkg="ir2425_group_08" type="node_b" output="screen" />
</launch>